docs/source/daml2ts/index.rst

```diff
@@ -267,6 +263,8 @@ What **is** important is the use of the companion objects when creating contract
    :linenos:
 
    import Ledger from  '@daml/ledger';
+   import * as '03d1a767741153b5910ad849b0f7af96d205416b3e0ec4c23cbc75a0177776f4/Iou';
```

Comment:
I don't think that's valid TypeScript. We should have something like `import * as Iou from ...`.

Reply:
✔️